diff --git a/test_secret_santa_generator.py b/test_secret_santa_generator.py
index da56fe2..4f5f2cb 100644
--- a/test_secret_santa_generator.py
+++ b/test_secret_santa_generator.py
@@ -3,7 +3,7 @@ from secret_santa_generator import SecretSantaGenerator
 
 
 def person_from(name, couple_id):
-    return {'name': name, 'couple_id': couple_id}
+    return {'name': name, 'couple_id': couple_id} # Some comment that's relevant
 
 
 def test_individuals():
